src/Base/Unit.cpp

```diff
@@ -572,3 +582,9 @@ Unit Unit::HeatFlux                    (0,1,-3,0,0);
 Unit Unit::DynamicViscosity            (-1,1,-1);  // SI unit: kg/m/s
 Unit Unit::KinematicViscosity          (2,0,-1);  // SI unit: m^2/s, https://en.wikipedia.org/wiki/Viscosity#Kinematic_viscosity
 Unit Unit::VacuumPermittivity          (-3,-1,4,2); // SI unit: A²*s⁴/kg/m³ https://en.wikipedia.org/wiki/Permittivity#Vacuum_permittivity
+Unit Unit::VolumeFlowRate              (3,0,-1);
+Unit Unit::DissipationRate             (2,0,-3);  // SI unit: m^2/s^3 https://cfd-online.com/Wiki/Turbulence_dissipation_rate
```

Comment:
I think the best is to just use  the Wiki link

Reply:
I couldn't find one, so I have removed the cfd-online link.

Reply:
Oh, my mistake you already correctly use the Wiki link I mean. So keeping
https://www.cfd-online.com/Wiki/Turbulence_dissipation_rate
is correct. Please keep it.

Reply:
OK, I have added it back.